app/templates/components/forms/admin/settings/images-form.hbs

```diff
@@ -78,7 +78,7 @@
       {{input type='number' name='logo_height' value=image.eventImageSize.logoHeight min=1}}
     </div>
   </div>
-  {{t 'Standard Size of the available area is 500px X 200px'}}
+  {{t 'Standard Size of the available area is 32px X 32px'}}
```

Comment:
where is this coming from?

Reply:
@CosmicCoder96 All these sizes are same, as already filled in the input fields. This format was same for all the other sections except this.